### DISCUSSION TITLE
Tempi di attesa per le firme.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#1192] [Supporto servizi cooperativi]
Buonasera,

potete indicarci quali sono i tempi di attesa per la firma digitale (presa visione), dei dichiaranti, dopo l'invio della mail, ed il tempo disponibile per la firma dell'USC dopo l'eventuale passaggio dell'evento allo stato di 'FIRMATO DAL DICHIARANTE', attualmente impostati in Produzione?

In attesa di riscontro,

cordiali saluti.

**Comment by acolosimosogei** [comment]
Buonasera,

i tempi di validità del link inviato via mail per la firma digitale dei dichiaranti è impostato a 60 minuti e il tempo disponibile per l'USC di recuperare l'atto in stato di 'FIRMATO DAL DICHIARANTE' è di 336 ore.

Grazie, procedo a chiudere la discussion. 
Assistenza ANSC
